tools/tf2openapi/types/tfmetagraph_test.go

```diff
@@ -0,0 +1,159 @@
+package types
+
+import (
+	"github.com/kubeflow/kfserving/tools/tf2openapi/generated/framework"
+	pb "github.com/kubeflow/kfserving/tools/tf2openapi/generated/protobuf"
+	"github.com/onsi/gomega"
+	"testing"
+)
+
+/* Expected values */
+func expectedTFMetaGraph() TFMetaGraph {
```

Comment:
ptr?

Reply:
I thought of this expected value as "immutable", so I made it an object